uber/templates/base.html

```diff
@@ -205,7 +150,39 @@
                             </ul>
                         </li>
                     </ul>
-                    <ul class="nav navbar-nav" id="main-menu"></ul>
+                    {% block mainmenu %}
+                    <ul class="nav navbar-nav" id="main-menu">
+                        {% block mainmenu_items %}
+                        {% for menu_item in c.MENU_JSON_DECODED %}
+                            {% if menu_item.href %}
+                                <li><a href="{{ menu_item.href }}">{{ menu_item.name }}</a></li>
+                            {% elif menu_item.submenu %}
+
+                                <li class="dropdown">
+                                    <a href="#" class="dropdown-toggle" data-toggle="dropdown">
+                                        {{  menu_item.name }}
+                                        <span class="caret"></span>
+                                    </a>
+
+                                    <ul class="dropdown-menu" role="menu">
+                                    {% for submenu_item in menu_item.submenu %}
+                                        <li>
+                                            {% if not submenu_item.href %}
```

Comment:
pretty sure you're right though.... how did I.... huh.  how was this working earlier on my box..... baffled :)  you think I would have noticed that nothing was working at all in the submenus : )